### PR TITLE
Seeds updates

### PR DESCRIPTION
**Pull request by tomecho**
changed the seeds file around, it now reflects the use of the test data and there are I created a bunch of extra items so developers can make mistakes without having to reset the database every time to remove a reservation.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -4,8 +4,16 @@
 service = Service.find_by url: 'localhost'
 unless service
   service = FactoryGirl.create :service, name: 'dev stuff', url: 'localhost'
-  item_type = FactoryGirl.create :item_type, name: 'TEST_CREATE_RENTAL_TYPE', creator: service
-  item = FactoryGirl.create :item, name: 'TEST_CREATE_RENTAL_ITEM', reservable: true, item_type: item_type
+  item_type = FactoryGirl.create :item_type, name: 'TEST_ITEM_TYPE', creator: service
+  another_item_type = FactoryGirl.create :item_type, name: 'TEST_ITEM_TYPE_2', creator: service
+  # now we're going to need a lot of items to test with
+  FactoryGirl.create :item, name: 'TEST_CREATE_RENTAL_ITEM', reservable: true, item_type: item_type
```
Comment (dfaulken) [inline db/seeds.rb]: So I'm confused, what about the factory isn't sufficient for generating items?

Comment (tomecho) [inline db/seeds.rb]: Oh thats right, we're using factory girl, I had forgotten that.

Comment (phantompatrol) [inline db/seeds.rb]: Side note: I know I am playing 'johnny come lately' here, but I not sure FG is the right tool for seed files.

Having seed files run in production is not unheard of. Worth discussion at some point.

File: db/seeds.rb [hunk 2]
```diff
@@ -4,8 +4,9 @@
 service = Service.find_by url: 'localhost'
 unless service
   service = FactoryGirl.create :service, name: 'dev stuff', url: 'localhost'
-  item_type = FactoryGirl.create :item_type, name: 'TEST_CREATE_RENTAL_TYPE', creator: service
-  item = FactoryGirl.create :item, name: 'TEST_CREATE_RENTAL_ITEM', reservable: true, item_type: item_type
+  item_type = FactoryGirl.create :item_type, name: 'TEST_ITEM_TYPE', creator: service
+  # now we're going to need a lot of items to test with
+  FactoryGirl.create :item, reservable: true, item_type: item_type until Item.count > 40
```
Comment (dfaulken) [inline db/seeds.rb]: Very minor stylistic opinion - I think it's easier to parse "do something 40 times" than "do this until there are 40 of something that is created in each iteration". That is, I might just use `40.times do` here.

**Pull request merged**